docs/production-deployment/cloud/billing-and-cost.mdx

```diff
@@ -30,49 +30,92 @@ tags:
   - invoice history
 ---
 
-The Account Owner and Finance Admin hold the ability to access and view detailed billing and cost information at any time, ensuring full transparency and control over financial aspects. If you need to assess your spending patterns, check invoice histories, or handle payment details, Temporal Cloud's Cost and Billing capabilities are able to provide the information you need.
+Temporal strives to provide full transparency over billing and costs.
+Account Owners and Finance Admins can view their [detailed billing information](https://cloud.temporal.io/billing) at any time.
+Use this information to assess your spending patterns, inspect your credit ledger, check your invoice histories, or update payment details as needed.
 
-For more information on Temporal Cloud pricing for actions, storage, and services/support, view [Pricing](/cloud/pricing).
+For more information on Temporal Cloud pricing for Actions, storage, and services/support, please visit our [Pricing](/cloud/pricing) page.
 
-On the **Billing** page, you can find the following details:
+Your [Billing](https://cloud.temporal.io/billing) page includes all or some of the following information, based on your plan:
 
-- [Current Balance](#current-balance): Your balance accrued thus far in the billing cycle
+- [Current balance](#current-balance): Your balance to date for this billing cycle
 - [Recent bill](#recent-bill): The amount of your most recent bill
-- [Invoice history](#invoice): A record of all past invoices
-- Credit information: If applicable, details about any credit grants, purchases, usage, and remaining credits.
+- [Invoices](#invoice): Access to all past invoices
+- [Credits](#credit-table): A record of all credit related transactions.
+  This includes details on credit grants, purchases, usage, and remaining credits, if applicable.
 
-If you are not on a standard plan, the Billing page may not include all of the information stated above. 
+Your [Usage](https://cloud.temporal.io/usage) provides an overview of your usage across your Namespaces.
```

Comment:
"Your Usage provides an overview of your usage across your Namespaces." reads weird to me.  Maybe something like:

The [Usage](https://cloud.temporal.io/usage) page provides an overview of your usage across your Namespaces.

Reply:
So I was talking on Zoom while doing this with Unni, and she wanted the two to be aligned "Your Usage" "Your Invoices", but she didn't seem to care _which_. So I can change both to "The".

Reply:
Aligned both spots to "The". Thanks.